meta-security-smack/lib/oeqa/runtime/smack.py

```diff
@@ -361,7 +361,7 @@ def test_smack_ambient(self):
         status, output = self.target.run(
             "echo '%s' > %s/ambient" %(initial_ambient, self.smack_path))
         self.assertEqual(
-            test_ambient, new_ambient.strip(),
+            test_ambient, ''.join(list(new_ambient)).strip(),
```

Comment:
It would have been more readable if you changed the new_ambient where it was defined, not in the assertion.